ledger/controllers/grants.js

```diff
@@ -89,6 +98,36 @@ const getPromotions = (protocolVersion) => (runtime) => async (request, reply) =
   reply(entries)
 }
 
+/*
+ GET /v3/promotions
```

Comment:
we want this logic in `GET /v3/grants` instead of `GET /v3/promotions` since that's what the browser uses